fleet_operations/report/daily_Parts_issuance_wizard_report.py

```diff
@@ -4,16 +4,12 @@
 import base64
 from datetime import datetime, timedelta
 from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from odoo import models
 
-try:
-    from odoo.addons.report_xlsx.report.report_xlsx import ReportXlsx
```

Comment:
Dont we need the alternative?

Reply:
Hello Sir,
In V11 report_xlsx module provide the report.report_xlsx.abstract model, we have to inherit that model in our report and call the generate_xlsx_report method to generate the .xlsx report. so no need to import it.